app/models/manageiq/providers/ibm_cloud/inventory/parser/vpc.rb

```diff
@@ -18,8 +18,8 @@ def parse
     auth_key_pairs
     flavors
     images
-    instances
     volumes
+    instances
   end
```

Comment:
why this change in the order?

Reply:
Probably because https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/76#discussion_r501380989 was using `.find()` so the volume had to be in the collection already.

With `.lazy_find()` the order shouldn't matter

Reply:
@chessbyte the instance needed to find its volume allocation and needed volumes built to find the size. As @agrare pointing out the order mattered because was using find vs lazy_find. I have switched to lazy_find and setting the order back. Thanks for the catch

Reply:
fixed